package.json

```diff
@@ -53,29 +53,34 @@
     "eslint-plugin-react": "^6.4.1",
     "eslint-plugin-standard": "^3.0.1",
     "jest": "^19.0.2",
-    "less": "^2.7.2",
-    "less-loader": "^4.0.5",
-    "patternfly": "^3.27.4",
+    "node-sass": "^4.7.2",
+    "patternfly": "^3.31.0",
     "prettier": "^1.7.4",
     "raf": "^3.4.0",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
     "react-test-renderer": "^16.0.0",
+    "sass-loader": "^6.0.6",
     "semantic-release": "^6.3.6",
     "style-loader": "^0.19.0"
   },
   "peerDependencies": {
-    "patternfly": "^3.27.4",
+    "patternfly": "^3.31.0",
```

Comment:
Yes, thank you.

Reply:
It shouldn't. Doing so breaks any possibility to customize patternfly styling

Reply:
After discussing this further, I agree patternfly needs to be a dependency since it is used to build compiled css. Although, a consumer application still needs to install patternfly too, which implies peer dependency.

Reply:
So as a consumer, i will need to `npm install --save patternfly patternfly-react`?

And import as?
```scss
$var-path: 'path';

@import '~patternfly-react/sass/patternfly-react';
```

Will it use my own installation of `patternfly`?

Reply:
You will need to import the patternfly partials as well as described in patternfly followed by the patternfly-react import.

Reply:
moved patternfly to normal dependencies.

Reply:
@jeff-phillips-18 fine w/ this...